src/Forms/Settings.cs

```diff
@@ -2553,5 +2558,18 @@ private void linkLabelOpenDictionaryFolder_LinkClicked(object sender, LinkLabelL
             System.Diagnostics.Process.Start(dictionaryFolder);
         }
 
+        private void textBoxVlcPath_MouseLeave(object sender, EventArgs e)
+        {
+            try
+            {
+                var path = textBoxVlcPath.Text.Trim('\"');
+                if (path.Length > 3 && Path.IsPathRooted(path) && Path.GetFileName(path).Equals("vlc.exe", StringComparison.OrdinalIgnoreCase) && File.Exists(path))
+                {

```

Comment:
I would assume, that might be the best guide for clueless users?
It wouldn't stop you, though, from entering something completely different. :smile: